lib/gcloud/dns/zone.rb

```diff
@@ -360,6 +363,7 @@ def records options = {}
           fail ApiError.from_response(resp)
         end
       end
+      alias_method :find_record, :records
```

Comment:
Rather than or in addition to this alias, I would love to have a `find_record_by name, type` convenience method that returns the first record found (or nil) rather than the array:

```ruby
record = zone.find_record_by "www", "A"
record.name #=> "www.example.com."
```